src/WebJobs.Extensions.OpenAI.Kusto/WebJobs.Extensions.OpenAI.Kusto.csproj

```diff
@@ -1,16 +1,27 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
-  <PropertyGroup>
-    <Description>Microsoft Azure WebJobs SDK Extension for OpenAI - Kusto Search Package.</Description>
-  </PropertyGroup>
-  
-  <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Functions.Extensions" Version="1.1.0" />
-    <PackageReference Include="Microsoft.Azure.Kusto.Data" Version="11.3.5" />
-    <PackageReference Include="Microsoft.Azure.Kusto.Ingest" Version="11.3.5" />
-  </ItemGroup>
+	<PropertyGroup>
+		<Description>Microsoft Azure WebJobs SDK Extension for OpenAI - Kusto Search Package.</Description>
+	</PropertyGroup>
+	<!-- Version settings: https://andrewlock.net/version-vs-versionsuffix-vs-packageversion-what-do-they-all-mean/ -->
+	<PropertyGroup>
+		<MajorVersion>0</MajorVersion>
+		<MinorVersion>13</MinorVersion>
+		<PatchVersion>0</PatchVersion>
+		<VersionPrefix>$(MajorVersion).$(MinorVersion).$(PatchVersion)</VersionPrefix>
```

Comment:
Properties such as `VersionPrefix`, `AssemblyVersion`, etc. are already defined in /src/Directory.Build.props. Do we have to redefine them here or can we just inherit those properties? Ideally we only need to explicilty define the major, minor, and patch versions in these csproj files.

Reply:
I tried building with just major, minor and patch versions, values in the prop take precendence in that case and whatever version is defined there gets created.

Reply:
I see. We might be able to instead define these properties in `Directory.Build.targets` instead of `Directory.Build.props` to remove the duplication. But this can be done separately if you prefer.